### ISSUE TITLE
TypeError: RN(...).setNotifier is not a function

### ISSUE THREAD

**Issue by dep** [honeybadger-io/honeybadger-js#1175]
## What are the steps to reproduce this issue?
1. install `"@honeybadger-io/react": "6.1.3"`

## What happens?
I'm getting this error on pageload:

```
TypeError: RN(...).setNotifier is not a function
```

## What were you expecting to happen?

No errors

## Any logs, error output, etc?

When I inspect this, it points to this code:

```
$o().setNotifier({
    name: "@honeybadger-io/react",
    url: "https://github.com/honeybadger-io/honeybadger-js/tree/master/packages/react",
    version: "6.1.3"
});
```

## Any other comments?

Any help is appreciated!

**Comment by dep** [comment]
Downgrading to `6.0.5` seemed to alleviate the problem for now

**Comment by BethanyBerkowitz** [comment]
Thanks for reporting this, @dep! 

I think the problem is likely an older version of a `@honeybadger-io/js`. Would you mind running `npm ls @honeybadger-io/js` and let us know what the version is? 

Could you also install the latest version of `@honeybadger-io/js` and let us know if that fixes it? If it does then there must be an issue with the peer dependency requirements, so we can go ahead and fix that.

**Comment by subzero10** [comment]
@dep, this should be fixed with `@honeybadger-io/react@6.1.5`!

cc @BethanyBerkowitz
